cmd/remoterun/test.go

```diff
@@ -59,7 +59,7 @@ commands:
 				return fmt.Errorf("--name is required")
 			}
 
-			oktetoContext, err := contextCMD.NewContextCommand().RunStateless(ctx, &contextCMD.Options{})
+			oktetoContext, err := contextCMD.NewContextCommand().RunStateless(ctx, &contextCMD.Options{Show: true})
```

Comment:
Why do we need to show the context the user is using in the remote? I checked and as it's not staged it's not showing but maybe I'm missing something

Reply:
mistake addressed here: 8f74637c130af204cb71b1e5abdca09428edb7d2